src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ArrayTrailingCommaCheck.java

```diff
@@ -68,12 +92,27 @@ public void visitToken(DetailAST arrayInit) {
 
         // if curlies are on the same line
         // or array is empty then check nothing
-        if (arrayInit.getLineNo() != rcurly.getLineNo()
+        if ((!ignoreInlineArrays || arrayInit.getLineNo() != rcurly.getLineNo())
             && arrayInit.getChildCount() != 1) {
             final DetailAST prev = rcurly.getPreviousSibling();
             if (prev.getType() != TokenTypes.COMMA) {
                 log(rcurly.getLineNo(), MSG_KEY);
             }
         }
     }
+
+    /**
+     * Set the ignoreInlineArrays to enforce.
+     *
+     * @param ignoreInlineArraysStringValue string to decode ignoreInlineArrays from
+     * @throws IllegalArgumentException if unable to decode
+     */
+    public void setIgnoreInlineArrays(String ignoreInlineArraysStringValue) {
+        try {
+            ignoreInlineArrays = Boolean.parseBoolean(ignoreInlineArraysStringValue);
```

Comment:
no need. We already have the converter.  Just use boolean argument.
FYI: https://github.com/checkstyle/checkstyle/blob/ce21086e087661553f3a774c38362327ee88996a/src/main/java/com/puppycrawl/tools/checkstyle/api/AutomaticBean.java#L89-L89